hc/api/tests/test_create_check.py

```diff
@@ -47,35 +48,66 @@ def test_it_works(self):
 
     def test_it_accepts_api_key_in_header(self):
         payload = json.dumps({"name": "Foo"})
-
-        ### Make the post request and get the response
-        r = {'status_code': 201} ### This is just a placeholder variable
-
-        self.assertEqual(r['status_code'], 201)
+        resp = self.client.post(self.URL, payload, HTTP_X_API_KEY="abc", 
+                content_type="application/json")
+        self.assertEqual(resp.status_code, 201)
 
     def test_it_handles_missing_request_body(self):
-        ### Make the post request with a missing body and get the response
-        r = {'status_code': 400, 'error': "wrong api_key"} ### This is just a placeholder variable
-        self.assertEqual(r['status_code'], 400)
-        self.assertEqual(r["error"], "wrong api_key")
+        resp = self.client.post(self.URL,content_type="application/json")
+        self.assertEqual(resp.status_code, 400)
+        self.assertEqual(resp.json()["error"], "wrong api_key")
```

Comment:
This test is for a missing request body  but the assert asserts for a wrong API key. Why is that?

Reply:
The api_key is supposed to be part of the request body. The error returned is wrong API key because the body is missing